scripts/create_release

```diff
@@ -0,0 +1,105 @@
+#! /usr/bin/env python3
+#
+# YacGuide Android Application
+#
+# Copyright (C) 2020 Christian Sommer
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Script for creating a releases."""
+
+import os
+import re
+import argparse
+from datetime import datetime
+import git
+
+
+def get_cli_parser():
+    """Return argument parser object."""
+    parser = argparse.ArgumentParser(description=__doc__)
+    help_release_type = "release type (default: %(default)s)"
+    parser.add_argument("--release-type", choices=["dev"],
+                        default="dev", required=False,
+                        help=help_release_type)
+    help_version = """release version string. For release type 'dev'
+it is the date code in the format 'YYYYMMDD'."""
+    parser.add_argument("--version", required=False,
+                        help=help_version)
+    help_skip_checks = "skip checks"
+    parser.add_argument("--skip-checks", action="store_true",
+                        default=False, required=False,
+                        help=help_skip_checks)
+    return parser
+
+
+class Release:
+    """Release Class
+
+    Attributes:
+        cli_parser (argparse.ArgumentParser): Command line interface
+            parser
+        cli_args (argparse.Namespace): Parsed CLI arguments
+        repo (git.Repo): Git repo object
+    """
+    _GRADLE_FILE = "app/build.gradle"
```

Comment:
Sounds like we must run this script from within a certain directory? However, it seems that this script should be only executed by the makefile. Please add an according information like ```Please do not run this script manually. Use make instead``` to the top of the file in that case

Reply:
It is not forbidden to run this script manually, if you know what you do. The Makefile wrapper is mainly necessary for convenience because all the scripts must run within the container. As we already discussed, I probably will replace the Makefile stuff by Python.